coldfront/core/group/management/commands/create_default_user_groups.py

```diff
@@ -0,0 +1,119 @@
+from django.core.management.base import BaseCommand
+
+from django.contrib.auth.models import Group, Permission
+
+DEFAULT_RIS_USERSUPPORT_GROUP_PERMISSIONS = [
+    {"id": 167, "codename": "add_allocation"},
+    {"id": 168, "codename": "change_allocation"},
+    {"id": 169, "codename": "delete_allocation"},
+    {"id": 170, "codename": "view_allocation"},
+    {"id": 171, "codename": "can_view_all_allocations"},
+    {"id": 172, "codename": "can_review_allocation_requests"},
+    {"id": 173, "codename": "can_manage_invoice"},
+    {"id": 174, "codename": "add_allocationattribute"},
+    {"id": 175, "codename": "change_allocationattribute"},
+    {"id": 176, "codename": "delete_allocationattribute"},
+    {"id": 177, "codename": "view_allocationattribute"},
+    {"id": 178, "codename": "add_allocationattributetype"},
```

Comment:
I don't think User Support needs to be able to add, change, delete allocation attribute *types*.

Similar story for attribute types and allocation and allocation user status choices.

Reply:
Done ✅